include/rocksdb/utilities/transaction_db.h

```diff
@@ -410,15 +410,17 @@ struct TransactionOptions {
   // When the number of updates in a transaction is at least this threshold,
   // we will enable optimizations for commiting a large transaction. See
   // comment for `commit_bypass_memtable` for more optimization detail.
-  uint32_t large_txn_commit_optimize_threshold =
-      std::numeric_limits<uint32_t>::max();
+  //
+  // Default: 0 (disabled).
+  uint32_t large_txn_commit_optimize_threshold = 0;
```

Comment:
nit: the "  //  large_txn_commit_optimize_byte_threshold to max." type of comment above needs some update.

Reply:
Fixed, thanks for catching this.